src/fides/api/task/execute_request_tasks.py

```diff
@@ -227,23 +227,32 @@ def run_access_node(
 
         if can_run_task_body(request_task):
             # Build GraphTask resource to facilitate execution
-            graph_task: GraphTask = create_graph_task(session, request_task)
-            # Currently, upstream tasks and "input keys" (which are built by data dependencies)
-            # are the same, but they may not be the same in the future.
-            ordered_upstream_tasks: List[
-                Optional[RequestTask]
-            ] = _order_tasks_by_input_key(
-                graph_task.execution_node.input_keys, upstream_results
-            )
-            # Pass in access data dependencies in the same order as the input keys.
-            # If we don't have access data for an upstream node, pass in an empty list
-            upstream_access_data: List[List[Row]] = [
-                upstream.get_decoded_access_data() if upstream else []
-                for upstream in ordered_upstream_tasks
-            ]
-            # Run the main access function
-            graph_task.access_request(*upstream_access_data)
-            log_task_complete(request_task)
+            with TaskResources(
+                privacy_request,
+                privacy_request.policy,
+                session.query(ConnectionConfig).all(),
+                request_task,
+                session,
```

Comment:
This mimics what DSR 2.0 is doing so we're closing connections to external databases after the request is done